internal/cloud/realm/access_list.go

```diff
@@ -12,6 +12,12 @@ const (
 	allowedIPsPathPattern = appPathPattern + "/security/allowed_ips"
 )
 
+// AccessList is a list of allowed IPs stored in a Realm app
+type AccessList struct {
+	AllowedIPs []AllowedIP `json:"allowed_ips"`
+	CurrentIP  string      `json:"current_ip"`
```

Comment:
[nit] I'd recommend just removing this field until its actually needed (similar to `realm.App` and how it only partially defines the fields it needs)